core/src/main/java/org/hisp/dhis/android/core/D2.kt

```diff
@@ -74,8 +74,7 @@ import org.hisp.dhis.android.core.wipe.internal.WipeModule
 class D2 internal constructor(internal val d2DIComponent: D2DIComponent) {
     private val modules: D2Modules = d2DIComponent.modules
 
-    @VisibleForTesting
-    internal fun ktor(): KtorServiceClient {
+    fun ktor(): HttpServiceClient {
```

Comment:
This method is exposed and is named `ktor`, shouldn't it be called `httpServiceClient`?